demo.sh

```diff
@@ -1,2 +1,7 @@
 this is demo.updated data here
-i am adding this for Pull Request
+i am adding this for Pull Request from UAT to master
+
+i have added this is extra line
```

Comment:
remove the empty line
remove the line number 4 to line number 7
```suggestion
i have added this is extra line
```